.github/vale/README.rst

```diff
@@ -0,0 +1,456 @@
+========================
```

Comment:
Please revisit this file and condense it. As it is, people can't read all of it. What information is needed:
 - if you don't know what Vale is
 - if you're tackling a build failure (probably how most people meet it)
 - for a maintainer dealing with some more advanced feature

Could some of this information be linked to on Vale's own docs, for example? My concern here is that we aren't publishing something useful to the audience, and it's at a level of depth that we won't be able to actively maintain without a lot of deep work. I think 800 words max would be a sensible goal.

Reply:
I definitely agree.
There are three things here:

1. The useful kernel of things people want to know, hidden (as you say) amongst:
2. Notes I made for myself as I was learning, and
3. Things that should be in the vale docs somewhere, but aren't yet.

The second and third need removing as they're not part of the job of this repository (well, a little info from 3 needs to be kept, for your second point, but not at this detail, and only at an "if this happens, here is how to fix it" level).

The "extra" text that gets removed can safely live in the git history, possibly with a tag to make it easy for me to find.

Reply:
There's now a much shortened vale README.
I *think* it's a good first try, and will be interested in whether I should cut more stuff.